django/test/channels.py

```diff
@@ -0,0 +1,61 @@
+from asgiref.inmemory import ChannelLayer as InMemoryChannelLayer
```

Comment:
This is making asgiref a hard dependency for Django (this file is imported from django.test), I think that should be avoided.